source/voice_control/create_wake_word.markdown

```diff
@@ -66,6 +68,44 @@ Depending on the word, training a model on your own wake word may take a few ite
 7. Test your new wake word.
    - Speak your wake word followed by a command, such as "Turn on the lights in the kitchen".
    - When the ATOM Echo picks up the wake word, it starts blinking blue.
+  
+### For a Wyoming-openWakeWord Docker Container
+
+To add custom models to a Wyoming-openWakeWord docker container, you need to add a new folder and tell Wyoming-openWakeWord where to look for custom models.
+
+1. Create a folder for custom models in a convenient location, for example alongside your data directory for openwakeword.
+   - For example: `mkdir ~/openwakeword/custom`
+2. Copy your custom models to this folder.
+3. If you're using docker compose, update your `compose.yaml` to add a volume to give access to this new folder within your docker container.
+   - Example `compose.yaml` configuration:
+     ```yaml
+     version: '3.3'
+     services:
+       wyoming-openwakeword:
+         restart: unless-stopped
+         volumes:
+           - '/replace/with/path/to/data:/data'
+           - '/replace/with/path/to/custom/models:/custom'
+         ports:
+           - '10400:10400'
+         image: rhasspy/wyoming-openwakeword
+         command: " --preload-model 'ok_nabu' --custom-model-dir /custom"
+     ```
```

Comment:
This seems like instructions for the upstream project, not Home Assistant.

Reply:
That's fair. I could just reduce the section to a general statement, and link to the upstream instructions for adding custom wake words within their docker setup (which I would likely submit improvements to as well separately).

The thing I'd like to make sure to include is some instruction for HA container, to avoid the confusion of having instructions only for HAOS. I was quite confused personally.